### DISCUSSION TITLE
Automatically delete rooms to save database space

### DISCUSSION THREAD

**Discussion by NigelRidderhof** [mozilla/hubs#5669] [Hubs Cloud Questions]
Hey! A long time ago I asked a question in the Discord regarding automatic deletion of rooms to prevent hundreds of created and now unused rooms to be out there taking up database space or such. 

I personally did not see anything about this come by in the previous months, so I was wondering if there has been any changes / progress regarding such a functionality in which rooms would be automatically deleted after f.e. a week of not being used. Garbage collection of some sort.

Maybe some new feature for this public hubs API I've seen posts about for a while?

**Comment by pattersonbl2** [comment]
I don't have this feature request on my radar right now. if you wouldn't mind opening a feature request issue for this,  This helps us track ideas for future changes.

**Reply by NigelRidderhof** [reply]
Check, here it is: https://github.com/mozilla/hubs/issues/5678

If it's the case that rooms barely cost anything, even if you have hundreds or thousands of them, then insights on this / places in AWS to check would be very much appreciated as well!
